### ISSUE TITLE
OSW test is failing 

### ISSUE THREAD

**Issue by anyaelena** [NREL/alfalfa#96]
Not sure what changed but now getting this.  I've commented out the test until you can look at it.  :cringe:
`INFO:alfalfa_worker:Calling step_sim_type subprocess: ['python3', 'step_sim/step_osm.py', '4e6937ae-4a1c-11eb-9c63-88e9fe5dee0a', 'timescale', '2019-01-02 00:00:00', '2019-01-03 00:00:00', '--step_sim_value=5']
INFO:botocore.credentials:Found credentials in environment variables.
Program terminated: EnergyPlus Terminated--Error(s) Detected.
/usr/local/bin/runenergyplus: line 603: /usr/local/bin/ReadVarsESO: No such file or directory
/usr/local/bin/runenergyplus: line 604: /usr/local/bin/ReadVarsESO: No such file or directory
/usr/local/bin/runenergyplus: line 612: /usr/local/bin/HVAC-Diagram: No such file or directory
{d58ea733-4aa4-4d0b-8784-543119c84621}current_day{ad15967d-ad30-4052-a032-d14b6d9c0b46}current_hour{d1e05687-c4cb-4aa4-96cd-8f6373a08d30}current_minute{fa00bf57-9d1c-4fc0-8a27-de96424b7bb5}current_monthNamespace(end_datetime='2019-01-03 00:00:00', site_id='4e6937ae-4a1c-11eb-9c63-88e9fe5dee0a', start_datetime='2019-01-02 00:00:00', step_sim_type='timescale', step_sim_value=5)
Server started on 127.0.0.1:48899
Wrote socket.cfg`

**Comment by corymosiman12** [comment]
Maybe relevant for this.  So I deployed Alfalfa locally, then accessed the `alfalfa_worker` shell.

I'm assuming it fails because it is unable to find the idf file (below):
```
root@b0121becfd58:/alfalfa/alfalfa_worker# pwd
/alfalfa/alfalfa_worker
root@b0121becfd58:/alfalfa/alfalfa_worker# ls /simulate/7eca424a-4f07-11eb-9d18-3af9d38d2919/simulation/
haystack_report_mapping.json  sim.epw                       sim.idf                       variables.cfg
root@b0121becfd58:/alfalfa/alfalfa_worker# ls /simulate/7eca424a-4f07-11eb-9d18-3af9d38d2919/simulation/sim.idf -li
2651745 lrwxrwxrwx 1 root root 98 Jan  5 03:38 /simulate/7eca424a-4f07-11eb-9d18-3af9d38d2919/simulation/sim.idf -> ../workflow/Users/cmosiman/Github/alfalfa/tests/integration/models/rtu_coordination_osw/run/in.idf
root@b0121becfd58:/alfalfa/alfalfa_worker# ls /simulate/7eca424a-4f07-11eb-9d18-3af9d38d2919/simulation/sim.epw -li
2651743 lrwxrwxrwx 1 root root 130 Jan  5 03:38 /simulate/7eca424a-4f07-11eb-9d18-3af9d38d2919/simulation/sim.epw -> ../workflow/Users/cmosiman/Github/alfalfa/tests/integration/models/rtu_coordination_osw/weather/USA_CO_Golden-NREL.724666_TMY3.epw
root@b0121becfd58:/alfalfa/alfalfa_worker# ls /simulate/7eca424a-4f07-11eb-9d18-3af9d38d2919/simulation/
haystack_report_mapping.json  sim.epw                       sim.idf                       variables.cfg
root@b0121becfd58:/alfalfa/alfalfa_worker# ls /simulate/7eca424a-4f07-11eb-9d18-3af9d38d2919/simulation/haystack_report_mapping.json -li
2651744 lrwxrwxrwx 1 root root 124 Jan  5 03:38 /simulate/7eca424a-4f07-11eb-9d18-3af9d38d2919/simulation/haystack_report_mapping.json -> ../workflow/Users/cmosiman/Github/alfalfa/tests/integration/models/rtu_coordination_osw/reports/haystack_report_mapping.json
root@b0121becfd58:/alfalfa/alfalfa_worker# cat /simulate/7eca424a-4f07-11eb-9d18-3af9d38d2919/simulation/sim.idf
cat: /simulate/7eca424a-4f07-11eb-9d18-3af9d38d2919/simulation/sim.idf: No such file or directory
```

I don't think this should be the expected behavior though, right?  Maybe I'm misunderstanding.

The other thing I noticed is that you are trying to access BCL measures (is this hitting the BCL web server?) https://github.com/NREL/alfalfa/blob/2c40ea5293fedf1882f939259143ced0cc613cef/alfalfa_worker/add_site/merge_osws.rb#L33

I would be hesitant of this as I know BCL is going through some updates now. Probs ask Nick / Katherine?

**Comment by kbenne** [comment]
thanks for the notes. getBCLMeasure is being used to find the measure locally in the osw directory.

**Comment by anyaelena** [comment]
We believe this is failing related to specific missing Python dependencies running alongside E+.  So that test is not expected to pass but we're working on a replacement osw test.

**Comment by anyaelena** [comment]
Fixed in #107
